app/src/main/java/org/simple/clinic/summary/PatientSummaryInit.kt

```diff
@@ -8,11 +8,7 @@ import org.simple.clinic.summary.OpenIntention.LinkIdWithPatient
 class PatientSummaryInit : Init<PatientSummaryModel, PatientSummaryEffect> {
 
   override fun init(model: PatientSummaryModel): First<PatientSummaryModel, PatientSummaryEffect> {
-    val effects = mutableSetOf<PatientSummaryEffect>()
-
-    if (!model.hasLoadedPatientSummaryProfile) {
-      effects.add(LoadPatientSummaryProfile(model.patientUuid))
-    }
+    val effects = mutableSetOf<PatientSummaryEffect>(LoadPatientSummaryProfile(model.patientUuid))
```

Comment:
Can you please specify as to why we needed to make this change?
If I understand correctly, it is just to avoid delays in loading the patient profile in this view or is it something else?
We were already taking care of showing edited patient info via screen key?

Reply:
If we edit the patient and come back to the patient summary screen, the details on the header do not get updated immediately because we restore the last saved model instead of fetching the latest.

A "correct" approach is to detect from a screen result whether the patient has been edited and then requery it, but it seemed like a lot of maintenance effort for a local database call so I decided to query the patient details again.